app/mesh/1.5.x/license.md

```diff
@@ -0,0 +1,115 @@
+​
+In multi-zone mode, Kong Mesh automatically synchronizes the license to the remote `kuma-cp`, therefore this operation is only required on the global `kuma-cp`.
+​
+## Licensed Metrics
+​
+The license encourages a pay-as-you-go model that delivers the best benefits to you, the end user, since the derived value of Kong Mesh is directly associated to the positive benefits of real service mesh usage.
+​
+These metrics are:
+​
+* Number of connected data plane proxies (DPPs), across every zone.
+* An expiration date that determines the duration of the license.
+​
+A data plane proxy (DPP) is a standard data plane proxy that is being deployed next to our services, either as a sidecar container or in a virtual machine. Gateway data plane proxies, zone ingresses and zone egresses are not counted.
+​
+To measure the number of data plane proxies needed in Kong Mesh we can use the following formula, that only applies to the services that we want to include in our service meshes:
+​
+```
+Number of DPPs = Number of Pods + Number of VMs.
+```
+​
+With a valid issued license, a data plane proxy will always be able to join the service mesh even if we go above the allowed limit in order to prevent service disruptions (a warning in the GUI and in the logs will identify this situation to the end user). On the other end, with the prebundled license, whenever we go over the maximum allowed number of DPPs the system will automatically refuse their connection.
+​
+## License API
+​
+The license can be inspected via the GUI or an API at the `/license` endpoint on the control plane. For example:
+​
+```
+$ curl <IP of the control plane>:5681/license
+{
+ "allowedDataplaneProxies": 20,
+ "activeDataplaneProxies": 2,
+ "expirationDate": "2032-11-09T00:00:00Z",
+ "demo": false
+}
+```
+​
+## Configuring the license
+​
+A valid license file can be passed to Kong Mesh in a variety of different ways.
+​
+### Kubernetes (kumactl)
+​
+When installing the Kong Mesh control plane with `kumactl install control-plane`, please provide a `--license-path` argument with a full path to a valid license file, for example:
+​
+```sh
+$ kumactl install control-plane --license-path=/path/to/license.json
+```
+​
+### Kubernetes (HELM)
+​
+To install a valid license via HELM, please follow these instructions:
+​
+1) Create a secret named `kong-mesh-license` in the same namespace where Kong Mesh is being installed. For example:
+​
+```sh
+$ kubectl create namespace kong-mesh-system
+$ kubectl create secret generic kong-mesh-license -n kong-mesh-system --from-file=/path/to/license.json
+```
+​
+Where `kong-mesh-system` is the namespace where Kong Mesh control plane is installed and `/path/to/license.json` is the path to a valid license file. Note that the file name should be `license.json` unless otherwise specified in `values.yaml`.
```

Comment:
```suggestion
Where:
* `kong-mesh-system` is the namespace where Kong Mesh control plane is installed
* `/path/to/license.json` is the path to a valid license file. The filename should be `license.json` unless otherwise specified in `values.yaml`.
```